### ISSUE TITLE
Change option naming - npage

### ISSUE THREAD

**Issue by KZRKZR** [ggPMXdevelopment/ggPMX#92]
npage = NULL ? 
Weird naming option -> to get all individual plots npage=ALL in pmx_report for individual plots

This does not seem to be passed through.
Do a test to see if the parameter has any effect.

**Comment by agstudy** [comment]
**n**page : **n** like number . Maybe better to have more explicit name. 
 We check in the code that napge is an integer or null : 

       assert_that(is_integer_or_null(npage))

NULL here is the default value of napge parameter. 
To plot all the plots you can either : 
   - set npage to NULL( it can be very hepful  in generic cases) 
  - or just omit it

**Comment by ggPMXdevelopment** [comment]
We would like to change the syntax to get all pages, because currently, it's not very intuitive (npage = NULL). I was thinking of something like npage = "all"?

**Comment by agstudy** [comment]
No. null is more coherent with the missing option ( missing or null plots all plots). Anyway it is a detail.

**Comment by KZRKZR** [comment]
Found issue in the scope of testing pmx_report with different values for npage:

```
  ctr <- theophylline()
  ctr %>% pmx_report(
    name="report23",
    save_dir = "E:/test",
    template="individual",
    format="both",
    npage=1
  )
```
return 

**Error: Plot(s) were registered within ggPMX but were not rendered. Footnotes may be wrong.** 

and in word report looks like:
[report23.docx](https://github.com/ggPMXdevelopment/ggPMX/files/4741780/report23.docx)

After this error, the previous calls, which work fine also return error:

```
  ctr %>% pmx_report(
    name="report23",
    save_dir = "E:/test",
    template="individual",
    format="both",
    npage=3:4
  )
```
**Error: Plot(s) were registered within ggPMX but were not rendered. Footnotes may be wrong.** 
At the same time, the report looks good.
Will create a separate issue.

**Comment by tynesjo** [comment]
Confirm `NULL` would be the conventional way of indicating that you don't want to limit the number of pages.
If an argument value like "all" were accepted then both integer and string classes would have to be accepted.

**Comment by biethbr1** [comment]
@tynesjo: I suggest to have the option "ALL" (user-friendly) even if it comes with the price of having two classes to be accepted

**Comment by tynesjo** [comment]
fixed by #130
